docs/readthedocs/source/doc/PPML/Overview/ppml.md

```diff
@@ -222,39 +222,177 @@ The result should look like this:
 
 #### 2.2.3 Run Trusted Big Data and ML on Cluster
 
-##### 2.2.3.1 Configure the Environment
+WARNING: If you want spark standalone mode, please refer to [standalone/README.md](https://github.com/intel-analytics/BigDL/blob/main/ppml/trusted-big-data-ml/python/docker-graphene/standalone/README.md). But it is not recommended.
 
-Prerequisite: [ssh login without password](http://www.linuxproblem.org/art_9.html) for all the nodes.
+Follow the guide below to run Spark on Kubernetes manually. Alternatively, you can also use Helm to set everything up automatically. See [kubernetes/README.md](https://github.com/intel-analytics/BigDL/blob/main/ppml/trusted-big-data-ml/python/docker-graphene/kubernetes/README.md).
+
+##### 2.2.3.1 Prepare data, key and password

```

Comment:
Merge 2.2.3.1 Prepare data & 2.2.3. 2K8S configurations into 2.2.3.1 Configure the Environment

Reply:
update in https://github.com/intel-analytics/BigDL/pull/4515/files#diff-206dc464459fee64731e43f5909c3e6144c71a8f9711eec547deb6ccf7e404e7R229